src/app/gov-analytics/components/DelegationCapitalDetails.tsx

```
@@ -0,0 +1,130 @@
+const ZERO = new BN(0);
+
+interface IDelegationCapitalDetails {
+	[key: string]: IGovAnalyticsDelegationStats;
+}
+
+function DelegationCapitalDetails({ delegationData }: { delegationData: IDelegationCapitalDetails }) {
+	const network = getCurrentNetwork();
+	const { resolvedTheme: theme } = useTheme();
+
+	const bnToIntBalance = (bnValue: string | number | BN): number => {
+		const bn = BN.isBN(bnValue) ? bnValue : new BN(bnValue && bnValue.toString());
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)